src/core/image.rs

```diff
@@ -75,13 +93,28 @@ where
     }
 }
 
-impl<T, C> Image<T, C>
+impl<T, C> ImageBase<T, C>
 where
+    T: Data,
+    C: ColourModel,
+{
+    /// Create an image given an existing ndarray
+    pub fn from_array(data: ArrayBase<T, Ix3>) -> Self {
```

Comment:
removed in another PR built on top of this since I realised I'd duplicated from_data